csharp/test/Microsoft.ML.OnnxRuntime.Tests/InferenceTest.cs

```diff
@@ -1653,6 +1657,214 @@ private void TestInferenceSessionWithByteArray()
             }
         }
 
+        void TestCPUAllocator(InferenceSession session)
```

Comment:
Fact ?

Reply:
Nope

---
In reply to: [463255195](https://github.com/microsoft/onnxruntime/pull/4646#discussion_r463255195) [](ancestors = 463255195)